app/views/providers/no_employment_incomes/show.html.erb

```diff
@@ -13,10 +13,10 @@
         <li><%= t '.details_3' %></li>
         <li><%= t '.details_4' %></li>
       </ul>
-      <p class="govuk-body"><%= t('.caseworker_will_ask') %></p>
-
+    
       <%= form.govuk_text_area :full_employment_details,
                                 label: {text: t('.employment_details_header'), tag: 'h2', size: 'm' },
+                                hint: { text: "You'll need to upload supporting evidence later." },
```

Comment:
Can this hint be moved into the local file too? Maybe as `employment_details_hint`?

Reply:
Good idea, I've made that change.